### DISCUSSION TITLE
Task 2 - Problem 4 b)

### DISCUSSION THREAD

**Discussion by rockyx50** [chulminy/CIVE497-CIVE700#119] [Q&A]
Hi there,

I'm a little confused with question 4 b). I found ao = 1, am =0 and bm = 0. I don't think this is correct since my Fourier series would just be 1/2 and I'm not sure where I went wrong. I have attached images of my integral below. Note I set Tp to 5 arbitrarily 

am = 
![image](https://user-images.githubusercontent.com/60040283/105940344-19efbe80-6029-11eb-824a-fadc5f92c30e.png)

bm =
![image](https://user-images.githubusercontent.com/60040283/105940333-0d6b6600-6029-11eb-89dd-f63398332d13.png)

**Comment by chulminy** [comment]
How about using this? 

![image](https://user-images.githubusercontent.com/3227103/105941244-fb8ac280-602a-11eb-8711-c9a9c2fa7ca1.png)


https://en.wikipedia.org/wiki/List_of_trigonometric_identities
